### ISSUE TITLE
AVT 1 - Carbon X Tabs Component DAP Violation 

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#1942]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed description 
Run DAP with the first tab, Tab Label 1 selected and DAP passes. (Note: Requested team to close issue #1833 because the DAP violation was gone now that there are no disabled Tabs.) but then found there were 4 DAP violations when focus moves to any of the other tabs 2 - 4 and DAP is run again (see screenshot):
<img width="1265" alt="screen shot 2019-02-27 at 2 14 54 pm" src="https://user-images.githubusercontent.com/21676914/53520506-6af08400-3a9b-11e9-8cac-59a0ffe2b248.png">

**Comment by dakahn** [comment]
Great catch. Thanks @snidersd

**Comment by snidersd** [comment]
Retested the tabs vanilla component and confirmed the DAP violations are still present. 
<img width="1280" alt="Screen Shot 2019-06-12 at 2 21 42 PM" src="https://user-images.githubusercontent.com/21676914/59379892-6df74380-8d1d-11e9-93a8-c1ae6bf46e07.png">
